src/main/java/domain/Operator.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (num1, num2) -> Integer.valueOf(num1 + num2)),
```

Comment:
이미 `BiFuntion`을 정의하실 때 `Integer`로 하셔서 `valueOf`를 하실 필요까지는 없어보입니다 😄 
추가적으로 그냥 알아두시면 좋을 것 같아서 😅 `Integer.valueOf`는 내부적으로 값을 캐싱하고 있습니다 🙏 
```java

    /**
     * Returns an {@code Integer} instance representing the specified
     * {@code int} value.  If a new {@code Integer} instance is not
     * required, this method should generally be used in preference to
     * the constructor {@link #Integer(int)}, as this method is likely
     * to yield significantly better space and time performance by
     * caching frequently requested values.
     *
     * This method will always cache values in the range -128 to 127,
     * inclusive, and may cache other values outside of this range.
     *
     * @param  i an {@code int} value.
     * @return an {@code Integer} instance representing {@code i}.
     * @since  1.5
     */
    @HotSpotIntrinsicCandidate
    public static Integer valueOf(int i) {
        if (i >= IntegerCache.low && i <= IntegerCache.high)
            return IntegerCache.cache[i + (-IntegerCache.low)];
        return new Integer(i);
    }
```

Reply:
헉,, 첨에 Double도 섞어서 쓰면서 실수로 추가했었나 봅니다.. 감사합니다!!